extension.driver.php

```diff
@@ -24,7 +24,7 @@ public function install()
 					`destination` VARCHAR(255) NOT NULL,
 					`validator` VARCHAR(50),
 					`unique` enum('yes','no') NOT NULL DEFAULT 'yes',
-					`default_main_lang` ENUM('yes', 'no') CHARACTER SET utf8 COLLATE utf8_unicode_ci NOT NULL DEFAULT 'no',
+					`default_main_lang` ENUM('yes', 'no', '') CHARACTER SET utf8 COLLATE utf8_unicode_ci NOT NULL DEFAULT 'no',
```

Comment:
Why is there a 3rd choice ? default_main_lang must be mandatory. Please remove the 3rd choice